cssom-view-1/Overview.bs

```diff
@@ -1246,15 +1249,25 @@ Note: The {{DOMRect}} object returned by {{Element/getBoundingClientRect()}} is
     1. If an ancestor of |this| in the [=flat tree=]
         has ''content-visibility: hidden'',
         return false.
-    1. If the {{CheckVisibilityOptions/checkOpacity}} dictionary member of |options|
-        is true,
+    1. If either the {{CheckVisibilityOptions/checkOpacity}}
+        or the {{CheckVisibilityOptions/opacityProperty}}
```

Comment:
I would list `opacityProperty` first, since it's the new name.

Reply:
Done.